LayoutKitSampleApp/OverlayViewController.swift

```diff
@@ -0,0 +1,95 @@
+// Copyright 2017 LinkedIn Corp.
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+import UIKit
+import LayoutKit
+import ExampleLayouts
+
+class OverlayViewController: UIViewController {
+
+    // MARK: - constants
+
+    // A smattering of alignments to test for the overlay and underlay views
+    private static let alignments: [Alignment] = [
+        .topLeading,
+        .topTrailing,
+        .topCenter,
+        .bottomLeading,
+        .bottomTrailing,
+        .bottomCenter,
+        .centerLeading,
+        .centerTrailing,
+        .center,
+        .fill
+    ]
+
+    // MARK: - variables
+
+    private var reloadableViewLayoutAdapter: ReloadableViewLayoutAdapter?
+    private var tableView = UITableView(frame: .zero, style: .grouped)
+    private var cachedLayouts: [Layout] = {
```

Comment:
`cachedLayouts` looks quite long as a variable. Do you think if that can break down?